packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkInfo.vue

```diff
@@ -37,7 +37,7 @@
         </oc-tag>
       </div>
       <div v-if="link.password">
-        <oc-tag>
+        <oc-tag class="oc-files-file-link-password">
```

Comment:
It is better to use `id` instead of `class` to identify an element if possible
```suggestion
        <oc-tag id="oc-files-file-link-password">
```

Reply:
I think you have it for consistency with others. So you can decide.